app/config.go

```diff
@@ -301,10 +316,12 @@ func InitKeyfiles() string {
 	if _, err := os.Stat(datadir + FS + GlobalConfig.TendermintConfig.PrivValidatorKey); err != nil {
 		// if not exist continue creating as other files may be missing
 		if os.IsNotExist(err) {
-			panic(fmt.Errorf("validator address not set! please run set-validator"))
+			fmt.Println(fmt.Errorf("validator address not set! please run set-validator"))
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatal("validator address not set! please run set-validator")

```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work